module/config.mjs

```diff
@@ -2554,6 +2554,9 @@ preLocalize("difficultTerrainTypes", { key: "label", sort: true });
  * @enum {MovementTypeConfig}
  */
 DND5E.movementTypes = {
+  walk: {
```

Comment:
If the order matters, should this object perhaps be frozen?

Reply:
That would prevent potentially customizing movement types (see https://github.com/foundryvtt/dnd5e/pull/4976).